README.md

```diff
@@ -93,6 +93,12 @@ No tests here...
 - GitHub: [@Salimer](https://github.com/Salimer)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/sbamahfoodh)
 
+👤 **Abdul Ali Noori**
+
+- GitHub: [@ab-noori](https://github.com/ab-noori)
+- Twitter: [@AbdulAliNoori4](https://twitter.com/AbdulAliNoori4)
+- LinkedIn: [abdulali-noori](https://www.linkedin.com/in/abdulali-noori)
+
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
I appreciate your recommendation. Actually I have my logo that I use in github and in my youtube channel. However, since it is a collaborative project I can't use my logo in this repo. I would appreciate your support in my youtube channel @HAMMAS-SALEEM 🥳

https://www.youtube.com/channel/UCGKq9eJQY2qTKRfDDbMOUyA

Reply:
Wow man I watched one of your videos, you are really producing high-quality videos 👍 Keep doing great things like these and keep shining brother 😊 May Allah guide you through the path 🤲

Reply:
@Salimer If you are working on a group project then it would be nice if you put all the authors logo otherwise if they don't have any then you can put yours only 😊